### PR TITLE
Add Siftery

### PR DESCRIPTION
**Pull request by TallGuysFreeADMIN**

### CODE REVIEW

File: data/siftery.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "id": "siftery",
+  "name": "Siftery",
+  "description": "Share products your company uses and get personalized recommendations.",
+  "categories": [],
+  "iconUrl": "https://pbs.twimg.com/profile_images/774378268950605824/9q92Ct1P_400x400.jpg",
+  "website": "https://www.siftery.com",
+  "countryHQ": "US",
+  "gdprReadyStatus": "nonCompliant",
```
Comment (Gdewilde) [inline data/siftery.json]: I'm currently awaiting a response from Siftery about their compliance status.

Comment (TallGuysFreeADMIN) [reply data/siftery.json]: I wouldn't wait much more than a week for a reply. They should know whether they are working towards it, or avoiding it. If they are GDPR compliant, they would likely announce it/update privacy policy :)

Comment (Gdewilde) [reply data/siftery.json]: I would mark this as in progress https://track.siftery.com/gdpr and would also update this to Siftery Track.

Comment (Gdewilde) [reply data/siftery.json]: Can you give me access to edit this PR?

**Pull request merged**